src/lando/main/models/repo.py

```diff
@@ -271,5 +302,6 @@ def apply_patch(self, patch_buffer: str):
     def last_commit_id(self) -> str:
         return self._run("rev-parse", "HEAD").stdout.strip()
 
-    def push(self):
+    def _native_push(self):
+        """XXX: unused?"""
```

Comment:
Let's remove this if unused.

Reply:
Actually, this may be something that's used, alongside the `_native_apply_patch` for push_to_try. It used to just be “repo.push()”, but I had to move it out of the way when I refactored the interface. The `_run` method it uses is git-only.

Reply:
Cool. I think this means that this code will be (re)moved for the moment, as I'm taking a different approach where this will be implemented in a `GitScm` object. I'll leave this here for now, with a clearer name, and I'll just move it when I get to the `GitScm` implementation.